game.js

```diff
@@ -42,8 +42,9 @@ class Game extends Screen {
 	}
 
 	/** Game initialization */
-	init() {
+	init(phrases) {
```

Comment:
As discussed, it doesn't make sense to initialize with a set of overriding phrases.